.idea/Fusion.iml

```diff
@@ -17,7 +17,7 @@
       <sourceFolder url="file://$MODULE_DIR$/FusionIIIT" isTestSource="false" />
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="jdk" jdkName="Python 3.8 (Fusion)" jdkType="Python SDK" />
+    <orderEntry type="inheritedJdk" />
```

Comment:
why did you make these changes?

Reply:
@akshatnema merge only those changes which are in the placement module section. Ignore the rest.

Reply:
So, please remove those changes from this PR. There is no specific option to merge manual changes from the PR.